test/providers.test.js

```diff
@@ -21,6 +21,19 @@ test('json rpc fetch block info', withProvider(async (provider) => {
     expect(sameBlock.header.height).toEqual(height);
 }));
 
+test('json rpc fetch gas price', withProvider(async (provider) => {
+    let stat = await provider.status();
+    let height = stat.sync_info.latest_block_height - 1;
+    let block = await provider.block(height);
+    let hash = block.header.hash;
+    let heightPrice = await provider.gasPrice(height);
+    let hashPrice = await provider.gasPrice(hash);
+    let lastestPrice = await provider.gasPrice(null);
+    expect(typeof heightPrice).toBe('object');
+    expect(typeof hashPrice).toBe('object');
+    expect(typeof lastestPrice).toBe('object');
```

Comment:
expect here to be a string that can be convert to an number, and that number > 0